chia/full_node/prometheus_full_node.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from typing import Dict
+
+from chia.util.prometheus import Prometheus
+
+# Default port for the full_node prometheus exporter
+DEFAULT_PROMETHEUS_PORT = 9914
+
+
+class PrometheusFullNode(Prometheus):
```

Comment:
Could we just have a `Prometheus` instance as an attribute instead of inheriting?  Also, it would be nice if all classes were `@dataclass`es.  In this case it looks like you would end up with a `create()` `@classmethod` or such to create a filled out instance.